lib/LaTeXML/resources/RelaxNG/LaTeXML-common.rng

```diff
@@ -408,7 +408,7 @@ Note that, unlike \TeX, this is the total height, including the depth (if any).<
 baseline of the content shown in the image.
 When displayed inilne, an image with a positive \attr{depth}
 should be shifted down relative to the baseline of neighboring material.</a:documentation>
-        <data type="integer"/>
+        <data type="float"/>
```

Comment:
Why?

Reply:
The depth is almost never an integer, and it goes into `vertical-align: NNpt;` so there is no harm in having decimals (and there is actual harm in  rounding it, as the SVG will be visibly off).

This applies to width/height as well. For scalable images, fractional sizes make sense and rounding causes artifacts. This is off topic for this PR, though.

Reply:
I'm generally a bit confused about possibly overloading and/or conflating things, here. `@imagedepth` goes with `@imagewidth,@imageheight` for images, whereas `@height,@depth,@width` are more generic, and probably are what should be used.  Is pgf responsible for setting `pgf@shift@baseline`?

Reply:
Yes, PGF sets `\pgf@shift@baseline` internally. I believe I used `@imagedepth` because it's the only relevant attribute that I had available for the task. I think. I admit I don't remember now.

Reply:
PS: I checked why I was using `@imagedepth`. This is because `<svg:svg>` has no `@depth` attribute, so the information has to go in `<ltx:picture>`, which already has the standard `@imagewidth` etc. I think that having *both* attributes on `<ltx:picture>` would be confusing.

Reply:
`imagedepth` is for image (files, usually raster), and is in pixels; I'm less worried at this point about fractional pixels. Either way, *something* is missing!

Reply:
I can get the right appearance with a simpler subset of what you've got here: Basically, in the afterDigest for `\lxSVG@insertpicture`, I set `style` to the vertical align based on what's in ``pgf@shift@baseline`.  So, then you don't need any extensions to schema or xslt. And that kinda makes sense, since `svg:svg` doesn't have a `depth` or `baseline` or `y-shift` or whatever attribute, preferring we use CSS. And on the other hand `ltx:picture` is a bit of a mess, having one foot in ancient pstricks code, and the other in modern svg.